src/EventGridExtension/EventGridExtensionConfig.cs

```diff
@@ -7,18 +7,30 @@
 using System.Threading.Tasks;
 using System.Web;
 using Microsoft.Azure.WebJobs.Host;
+using Microsoft.Azure.WebJobs.Host.Bindings;
 using Microsoft.Azure.WebJobs.Host.Config;
 using Microsoft.Azure.WebJobs.Host.Executors;
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
 
 namespace Microsoft.Azure.WebJobs.Extensions.EventGrid
 {
+
+    public class ReferenceOpenType : OpenType
```

Comment:
Can we use PocoOpenType from the SDK? Or if we're adding a new concept here, should we move this concept to the SKD?

Reply:
[`public class Poco : OpenType`](https://github.com/Azure/azure-webjobs-sdk/blob/f70476485521b73f9023ba3b46d8baed5157173c/src/Microsoft.Azure.WebJobs.Host/Bindings/OpenType.cs#L390) only exists in `dev` branch?